charts/nr-k8s-otel-collector/collector.md

```diff
@@ -243,7 +243,7 @@ kubeletstats:
 ```
     kubeletstats:
         collection_interval: {{ .Values.receivers.kubeletstats.scrapeInterval }}
-        {{- if include "newrelic.common.privileged" . }}
+        {{- if not (include "newrelic.common.gkeAutopilot" .) }}
```

Comment:
What's this for?

Reply:
The term `gkeAutopilot` is too overloaded, meaning we're using this for way more than should be intended.  Which I'd say the same for `privileged`, because this has nothing to do with that.

We should either use some other include (yet to be defined) just use something like `secureKubeletPort` or __PLACE_BETTER_NAME_HERE__ .  AK has similar features, as does EKS, and can be enabled (disabled by default since v1.32 in GKE with or without autopilot) on self managed clusters or disabled in AKS, GKE, prob more.

It's also recommended to disable the read only port.

Anyway, it's already in *common*, so we can just merge anyway without changing the name, since it is just a name, or create a new PR on common to add the include